applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1666,9 +1683,6 @@ void SmallStrainUPwDiffOrderElement::CalculateKinematics(ElementVariables& rVari
     noalias(rVariables.DNu_DX) = rVariables.DNu_DXContainer[GPoint];
```

Comment:
This now seems redundant. It was input for the B computation, which now already has taken place.